sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -808,6 +809,9 @@ object FunctionRegistry {
     expression[LengthOfJsonArray]("json_array_length"),
     expression[JsonObjectKeys]("json_object_keys"),
 
+    // Variant
+    expression[ParseJson]("parse_json"),
```

Comment:
Could we implement parse_json with another PR?

Reply:
I will have an actual implementation for `parse_json` in the future. At this point, I think it is beneficial to include a "fake" implementation to help testing and experimenting.